app/test/frontend/golden/analysis_0.4.0_http.json

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
To see changes in these files (i.e. diff's) it would be better if the filename was not including the version. Then we could easily see the small difference in the json file with changing pana versions.